lib/site_setting_extension.rb

```diff
@@ -554,12 +559,17 @@ def setup_methods(name)
           return false if !plugin.configurable? && plugin.enabled_site_setting == name
         end
 
-        if (c = current[name]).nil?
-          refresh!
-          current[name]
-        else
-          c
+        value =
+          if (c = current[name]).nil?
+            refresh!
+            current[name]
+          else
+            c
+          end
```

Comment:
```suggestion
        value =
          if current[name].nil?
            refresh!
            current[name]
          else
            current[name]
          end
```

I feel like this would be so much clearer than the random `c` (I know this is the old code :) )

Reply:
Or even better actually:

```ruby
refresh if current[name].nil?
value = current[name]
```

Reply:
ใช่ไหม